### PR TITLE
Sass 684

### PR DESCRIPTION
**Pull request by BenCosfordHMRC**
https://jira.tools.tax.service.gov.uk/browse/SASS-677

##### Before Reviewing
 - [x]  Have you pulled the branch down ?
 - [x]  Have you assigned yourself to the PR ? 
 - [x]  Have you moved the task to “in review” on JIRA ? 
 - [x]  Have you checked to ensure all dependencies are up to date ? 

##### Whilst Reviewing
 - [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
 - [x]  Have you checked for merge conflicts ?
 - [x]  Have you checked to make sure there are no builds in the pipeline before you merge? 
 - [x]  Have you moved the task to “in pipeline” on Jira ?

##### Before creating PR 
 - [x]  Have you addressed warnings where appropriate? 

##### After PRs been Reviewed
 - [x]  Have you checked the PR Builder passes ?
 - [x]  Have you checked for merge conflicts ?
 - [x]  Have you checked code coverage isn’t lower than previously ?

### CODE REVIEW

**Review by ina-tu — CHANGES_REQUESTED** [review]
The comments are from the terminal when I ran the tests. I'm not sure if these changes need to be made or not

File: conf/application.conf [hunk 1]
```diff
@@ -45,17 +45,12 @@ play.http.errorHandler = "config.ApiErrorHandler"
 # Play Modules
 # ~~~~
 # Additional play modules can be added here
-
-play.modules.enabled += "play.modules.reactivemongo.ReactiveMongoHmrcModule"
-
+play.modules.enabled += "uk.gov.hmrc.mongo.play.PlayMongoModule"
 play.modules.enabled += "config.ModuleBindings"
```
[the remark this answers is not in the document]
Comment (BenCosfordHMRC) [reply conf/application.conf:50]: Updated

[the remark this answers is not in the document]
Comment (BenCosfordHMRC) [reply conf/application.conf:50]: Updated

File: test/controllers/UserDataControllerSpec.scala [hunk 2]
```diff
@@ -36,8 +37,12 @@ class UserDataControllerSpec extends TestSupport with MockFactory with Results {
```
Comment (ina-tu) [inline test/controllers/UserDataControllerSpec.scala:37]: "java.lang.RuntimeException: Could not connect to mongo" appeared for the tests returning an internal server error tests. Is that right? The tests still passed

**Review by ina-tu — CHANGES_REQUESTED** [review]
"Logger configuration in conf files is deprecated and has no effect. Use a logback configuration file instead."

"No server chosen by com.mongodb.internal.async.client.ClientSessionHelper"

A few messages that showed up in the terminal for each test. I'm not sure about the second one

File: test/controllers/UserDataControllerSpec.scala (hunk 2 above)
Comment (BenCosfordHMRC) [reply test/controllers/UserDataControllerSpec.scala:37]: that's expected functionality

**Comment by BenCosfordHMRC** [comment]
> "Logger configuration in conf files is deprecated and has no effect. Use a logback configuration file instead."
> 
> "No server chosen by com.mongodb.internal.async.client.ClientSessionHelper"
> 
> A few messages that showed up in the terminal for each test. I'm not sure about the second one

logger config warning is a platform level issue I think
https://jira.tools.tax.service.gov.uk/browse/BDOG-798

**Review by ina-tu — APPROVED** [review]

**Pull request merged**